client/logic/scripts/ui/lobby.gd

```diff
@@ -60,8 +60,8 @@ func _on_button_back_pressed() -> void:
 
 func _on_button_start_pressed() -> void:
 	randomize()
-	var seed: int = randi()
-	var data: String = JSON.stringify({"seed": seed})
+	var rand_seed: int = randi()
+	var data: String = JSON.stringify({"seed": rand_seed})
```

Comment:
the random seed should not be provided by the hakama?

Reply:
In the end, yes, currently Nakama does only serve a relay role. Therefore, the creator of the lobby sets the seed.